packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchModalOverlay.vue

```diff
@@ -166,19 +166,19 @@ export default {
     background-color: transparent;
     padding: 0;
     margin: 0 0 0 25px;
-    color: $color-link-default;
+    color: $color-content-link;
     font-weight: $font-weight-bold;
     @include font-size(body-s, false);
     font-family: $font-family-base;
 
     &:hover {
         cursor: pointer;
-        color: $color-link-hover;
+        color: darken($color-content-link, $color-hover-01);
         background-color: transparent;
     }
     &:active,
     &:focus {
-        color: $color-link-active;
+        color: darken($color-content-link, $color-active-02);
```

Comment:
I would also make this `active-01` as active-02 is only to be used in combination with hover-02 (looking at the pie design token descriptions)